packages/scripts/src/upgrade-tests/voting2/2_Verify.ts

```diff
@@ -53,7 +55,7 @@ async function main() {
   const oldVoting = await getContractInstance<VotingEthers>("Voting", process.env[OLD_CONTRACTS.voting]);
   const proposer = await getContractInstance<ProposerEthers>("Proposer", process.env[OLD_CONTRACTS.proposer]);
 
-  const votingV2 = await getContractInstance<VotingEthers>("Voting", process.env[NEW_CONTRACTS.voting]);
```

Comment:
this did not affect anything due to the interfaces checked here being the same, right?

Reply:
Previously interfaces checked were the same, but now as we add check on `previousVotingContract` that is available only in v2 we need `VotingV2` interface.